src/main/java/de/tum/cit/aet/artemis/core/config/conditions/ArtemisConfigHelper.java

```diff
@@ -0,0 +1,47 @@
+package de.tum.cit.aet.artemis.core.config.conditions;
+
+import static de.tum.cit.aet.artemis.core.config.Constants.PROFILE_JENKINS;
+
+import org.springframework.core.env.Environment;
+import org.springframework.core.env.Profiles;
+
+/**
+ * Helper class for property configuration, in particular for determining conditions
+ * whether services should be enabled or not.
+ * This bridges the gap between the condition classes and the actual property values.
+ * WIP: make methods non-static
+ */
+public class ArtemisConfigHelper {
```

Comment:
We might want to discuss whether we directly use this class in the respective `Condition`s or if we want to create a new layer of abstraction for when using from within the initialized spring context. (like `ArtemisConfigServer` with @Service)

A reason could be to not be independent on the actual logic, but could also mock it etc.